src/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types'
+import { StyledCheckbox } from './Checkbox.styles'
+
+export function Checkbox({ disabled, checked }) {
+  return (
+    <StyledCheckbox
+      disabled={disabled}
+      defaultChecked={checked}
+      type="checkbox"
+      data-testid="checkbox"
+    />
```

Comment:
you can simply spread props on `StyledCheckbox`

```
export function Checkbox(props) {

return(
  <StyledCheckbox
    data-testid="checkbox"
    {...props}
    type="checkbox" // set the type here, then it can not be overridden by props object 
  />
)
```